src/app/contact-page/contact-page.component.html

```diff
@@ -0,0 +1,22 @@
+<app-title-bar></app-title-bar>
+
+<div id="form-container">
```

Comment:
We should keep styles consistent.  Instead of doing this, use `class="box"`, and wrap it in another div with `class="page-content"`.